### ISSUE TITLE
fill_in contenteditable element

### ISSUE THREAD

**Issue by dipth** [keathley/wallaby#129]
What is the procedure for filling out a field on a page that uses a wysiwyg editor instead of a plain textfield or area?

Specifically the page I'm trying to test is using the Trumbowyg editor which works by creating a div with the `contenteditable` attribute.
http://alex-d.github.io/Trumbowyg/

**Comment by dipth** [comment]
Here is an example of what the editor looks like in the DOM:

```html
<div class="trumbowyg-editor trumbowyg-reset-css" contenteditable="true" dir="ltr" style="height: 289px;"><p><span style="background-color: initial;">Here is my text</span></p></div>
```

Here is what I've tried to put in my test:
```elixir
session
|> fill_in(".trumbowyg-editor", with: "Testing 123...")
```

Here is the error that I'm getting:
```
(Wallaby.QueryError) Could not find any visible text input or textarea that matched: '.trumbowyg-editor'
```

**Comment by keathley** [comment]
Sorry about that, I haven't tried this with `fill_in`. You can try clicking on the element and then calling `[send_keys](https://hexdocs.pm/wallaby/Wallaby.Session.html#send_keys/2)` with your text.

**Comment by dipth** [comment]
I couldn't make that work so I instead used `execute_script` to set the inner html of the editor instead.

**Comment by keathley** [comment]
sorry about that @dipth. This should be fixed with the new api changes in #130.

**Comment by joepstender** [comment]
For anyone who tries to solve this: things indeed now work using send_keys, eg when testing a text-field for ck-editor. We have to fill both the invisible text_field and the div containing the editor:

```elixir
    session
    |> fill_in(text_field("procedure[text]", visible: false), with: "New procedure text")
    |> click(css("div.ck-editor__editable"))
    |> Wallaby.Browser.send_keys(["New procedure text"])
```
👍
